qtpy/QtCharts.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © 2019- The Spyder Development Team
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+"""Provides QtChart classes and functions."""
+
+# Local imports
+from . import PYQT5, PYSIDE2, PythonQtError
+
+if PYQT5:
+    from PyQt5 import QtChart as QtCharts
```

Comment:
> How do you want me to do it? Simple Try-catch?

Yes, please add a try/except here, with the except part returning a PythonQtError mentioning that you need to install QtCharts separately for PyQt5.